pages/shared/data/tools.yaml

```diff
@@ -466,3 +466,14 @@ providers:
       fit: contain
     formats:
       - email
+  - name: WompMobile
+    url: https://www.wompmobile.com
```

Comment:
Could you please link to a specific product/tool. Otherwise it's a bit confusing if you're listed under tools.

Reply:
Updated URL to point to relevant tool: https://www.wompmobile.com/amp-experience-platform/